proto/osmosis/concentrated-liquidity/incentive_record.proto

```diff
@@ -13,16 +14,10 @@ option go_package = "github.com/osmosis-labs/osmosis/v16/x/concentrated-liquidit
 // are included in the key so we avoid storing them in state, hence the
 // distinction between IncentiveRecord and IncentiveRecordBody.
 message IncentiveRecord {
-  uint64 pool_id = 1;
-  // incentive_denom is the denom of the token being distributed as part of this
-  // incentive record
-  string incentive_denom = 2
-      [ (gogoproto.moretags) = "yaml:\"incentive_denom\"" ];
-  // incentiveCreator is the address that created the incentive record. This
-  // address does not have any special privileges – it is only kept to keep
-  // incentive records created by different addresses separate.
-  string incentive_creator_addr = 3
-      [ (gogoproto.moretags) = "yaml:\"incentive_creator_addr\"" ];
```

Comment:
Note: as stated in the PR description, denom is moved to `IncentiveRecordBody` as part of `remaining_coin` and address is removed since, with the gauge infrastructure, the creator is always the incentives module address